packages/controllers/organizations.coffee

```diff
@@ -0,0 +1,11 @@
+if Meteor.isClient
+  Template.organizations.onCreated ->
+    @subscribe('currentUserOrganizations')
+
+  Template.organizations.helpers
+    organizations: ->
+      Organizations.find({createdById: Meteor.userId()})
+
+if Meteor.isServer
+  Meteor.publish 'currentUserOrganizations', ->
+    Organizations.find({createdById: this.userId})
```

Comment:
I really want to find a way to avoid hard-coding the same query in both the subscription and the publication.